.taskcluster.yml

```diff
@@ -147,6 +147,36 @@ tasks:
           owner: cdenizet@mozilla.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
+      - taskId: {$eval: as_slugid("check_submodules")}
+        dependencies:
+           - {$eval: as_slugid("lint_test_task")}
+           - {$eval: as_slugid("windows_test_task")}
+        created: {$fromNow: ''}
+        deadline: {$fromNow: '1 hour'}
+        provisionerId: proj-relman
+        workerType: ci
+        payload:
+          maxRunTime: 3600
+          image: "rust:buster"
+          command:
+            - "/bin/bash"
+            - "-cx"
+            - "git clone --recursive --quiet ${repository} &&
+               cd rust-code-analysis &&
+               git -c advice.detachedHead=false checkout --recurse-submodules ${head_rev} &&
+               rustup component add rustfmt &&
+               ./check-submodules.sh"
+          artifacts:
+            public/json-diffs-and-minimal-tests.tar.gz:
+              expires: {$fromNow: '2 weeks'}
+              path: /tmp/json-diffs-and-minimal-tests.tar.gz
+              type: file
+        metadata:
+          name: rust-code-analysis check submodules
+          description: rust-code-analysis check submodules
+          owner: cdenizet@mozilla.com
+          source: ${repository}/raw/${head_rev}/.taskcluster.yml
```

Comment:
Let's add a cache as we do for the `test_mozilla_central` task

Reply:
Cache didn't work the last time

Reply:
Do you mean it doesn't work in `test_mozilla_central`?

Reply:
Exactly, there were problems with Taskcluster

Reply:
OK, let's add it anyway as it's just two lines here, then I'll figure out why it's not working (let's file an issue about it so I don't forget)